src/common/legacy_config_opts.h

```diff
@@ -462,6 +462,7 @@ OPTION(mds_debug_subtrees, OPT_BOOL)
 OPTION(mds_kill_mdstable_at, OPT_INT)
 OPTION(mds_kill_export_at, OPT_INT)
 OPTION(mds_kill_import_at, OPT_INT)
+OPTION(mds_kill_dirfrag_at, OPT_INT)
```

Comment:
The reason you had build errors is because you're using the old lookup mechanism:

`g_conf()->foo`

If the option is removed from this file (and eventually they all will be), then you must do:

`g_conf().get_val<int64_t>("mds_kill_dirfrag_at");`

grep around for other examples.

Please also cache the value as these lookups can be expensive in hot paths. See 6837bf5e5242631292462efe2b3599ff8ebe0d8e for an example.

Reply:
Ahh thanks! Yea caching it is a good idea.